DNN Platform/Modules/MemberDirectory/Services/MemberDirectoryController.cs

```diff
@@ -138,7 +138,7 @@ private IEnumerable<UserInfo> GetUsers(int userId, int groupId, string searchTer
                     }
                     if (CanViewGroupMembers(portalId, groupId))
                     {
-                        users = UserController.Instance.GetUsersAdvancedSearch(portalId, userId, -1,
+                        users = UserController.Instance.GetUsersAdvancedSearch(portalId, PortalSettings.UserId, userId,
```

Comment:
The filterUserId parameter only used when search type is "Relationship", so IMO it can keep use -1 here.